### ISSUE TITLE
ASK-CLI 1.4 creates node 6 lambda functions

### ISSUE THREAD

**Issue by sem4phor** [alexa/alexa-skills-kit-sdk-for-nodejs#426]
## I'm submitting a...

<pre><code>
[ x ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

I know this isnt related to the ask directly but the ask-cli npm package has no github board...
ASK-CLI 1.3.* used to create lambda functions with node 8.* and the newest version 1.4 creates lambdas with 6.* which is problematic when you use some EcmaScript features in your code. When i change the lambda manually to 8 in the lambda console, i cant deploy anymore because the "revisionId" differs...

## Expected Behavior
Lambda functions are created with node 8 in ask cli v1.4

## Current Behavior
Lambda functions are created with node 6 in ask cli v1.4

## Possible Solution
```
// Not required, but suggest a fix/reason for the bug,
// or ideas how to implement the addition or change
```
Reach out to the ask-cli dev team and ask to change that.

## Steps to Reproduce (for bugs)
```
// Provide a self-contained, concise snippet of code
// For more complex issues provide a repo with the smallest sample that reproduces the bug
// Including business logic or unrelated code makes diagnosis more difficult
```
Deploy a skill with ask cli

## Context

I wanted to use EcmaScript features supported in node 8 but not in node 6

## Your Environment

* ASK SDK for Node.js used: 2.0.4
* Operating System and version: MacOS latest

## Node.js and NPM Info
* Node.js version used for development: 6
* NPM version used for development: 6.1.0

**Comment by tianrenz** [comment]
Hi @sem4phor , 

This issue is non-SDK related. We would recommend you to post in Alexa Developer Forum (see [here](https://forums.developer.amazon.com/spaces/165/alexa.html)) to get a more prompt response. 

Meanwhile, I've forwarded your request to the service owner team. 

Regards,

**Comment by RonWang** [comment]
Hi @sem4phor ,

We have just released a new version ask-cli@1.4.1. This version will catch the runtime issue you mentioned above and displays the instruction on how to trouble shoot. We also changed our code in the latest version to make sure all the lambda function is created using runtime nodejs8.10.

Thanks,
Ron
